### PR TITLE
Modbus, configuration without entities

### PR DESCRIPTION
**Pull request by janiversen**
## Proposed change

The modbus integration do not allow a configuration of a device without at least a single entity.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
https://github.com/home-assistant/core/pull/113516
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/modbus.markdown [hunk 1]
```diff
@@ -339,10 +339,16 @@ modbus:
 
 modbus entities are grouped below each modbus communication entry.
 
-All modbus entities have the following parameters:
+**REMARK** Each modbus device must have at least 1 entity defined.
+
+Experience have shown that a modbus configuration without entities leads to instability issues, which in worst case impact HA negatively.
```
Comment (davet2001) [inline source/_integrations/modbus.markdown]:
```suggestion
Experience has shown that a modbus configuration without entities leads to instability issues.
```
Grammar correction and suggest removing last part of a sentence which seems redundant.

File: source/_integrations/modbus.markdown [hunk 2]
```diff
@@ -339,10 +339,16 @@ modbus:
...
+
+The normal use case for the integration is to have entities, however there are some special cases and custom_components that used the modbus integration without entities. The solution to these special cases is to add a dummy entity.
```
Comment (davet2001) [inline source/_integrations/modbus.markdown]:
```suggestion
The normal use case for the integration is to have entities, however there are some special cases and custom components that used the modbus integration without entities. The solution to these special cases is to add a dummy entity.
```

**Review by davet2001 — CHANGES_REQUESTED** [review]

**Review by davet2001 — COMMENTED** [review]
The text looks good to me now. 

Is there a way we can prevent the configuration from being created without entities, to stop this from occurring rather than relying on documentation which may or may not be read/understood?

**Comment by janiversen** [comment]
The Pr in core, which is already merged, takes care of that.

**Comment by janiversen** [comment]
It is referenced above, if you want to check.

**Comment by janiversen** [comment]
Rebaased to newest Next (seems there was a couple of dependaboot changes, that potentially could affect the PR).

**Comment by janiversen** [comment]
@davet2001 you forgot to approve, of course unless you have additional changes.

**Comment by davet2001** [comment]
Ok understood, good that it's now already blocking the situation you want to avoid.

In that case the documentation could be a bit clearer.

I read it as though the user needed to take special care to avoid a problem.

Perhaps something like:

"The integration requires at least one valid entity for Home Assistant to operate reliably, and will be automatically blocked if none are configured.

For special cases such as use by custom components, please add a dummy entity to allow the integration to load, as prompted in the settings panel."

File: source/_integrations/modbus.markdown [hunk 3]
```diff
@@ -339,10 +339,16 @@ modbus:
 
 modbus entities are grouped below each modbus communication entry.
 
-All modbus entities have the following parameters:
+**REMARK** Each modbus device must have at least 1 entity defined, otherwise the integration will not be loaded.
+
+Experience has shown that a modbus configuration without entities leads to instability issues.
+
+The normal use case for the integration is to have entities, however there are some special cases and custom components that used the modbus integration without entities. The solution to these special cases is to add a dummy entity.
```
Comment (davet2001) [inline source/_integrations/modbus.markdown]:
```suggestion
The normal use case for the integration is to have entities, however there are some special cases and custom components that use the modbus integration without entities. The solution to these special cases is to add a dummy entity.
```

**Review by davet2001 — CHANGES_REQUESTED** [review]
Looks good now, just spotted one minor typo. Thanks!

File: source/_integrations/modbus.markdown (hunk 3 above)
Comment (janiversen) [reply source/_integrations/modbus.markdown]: Removed all the explaining, which really do not bring anything. The remark states clearly it is not allowed, and will prohibit the integration from loading.

**Review by davet2001 — APPROVED** [review]

**Pull request merged**